plugins/action/zos_copy.py

```diff
@@ -150,14 +160,59 @@ def run(self, tmp=None, task_vars=None):
                             dict(src=src, dest=dest, changed=False, failed=True)
                         )
                         return result
+
+                    if use_template:
+                        try:
```

Comment:
I think try block should only be needed when using template not in variable assignment. I suggest to use the block from line 172 to line 179.

Reply:
I moved down the try block. It still includes the creation and assigment of the `TemplateRenderer` object because its `__init__` method can throw two exceptions.

Reply:
Got it!